app/controllers/comments_controller.rb

```diff
@@ -11,6 +11,12 @@ def destroy
     redirect_to articles_path(@article)
   end
 
+  def show
+    @article = Article.find(params[:article_id])
```

Comment:
create a `before_action` filter with name of `find_article` and put that line in that method